src/inverted_index.c

```diff
@@ -619,10 +613,6 @@ DECODER(readNumeric) {
       res->num.value = INFINITY;
     } else if (header.encFloat.isDouble) {
       Buffer_Read(br, &res->num.value, 8);
-    } else {
```

Comment:
Maybe we want to keep it, for if we meet a float in the woods.

Reply:
returned code. add `TODO` instead